### PR TITLE
Macro: #3819 - Selection in sequence representation for view mode

### PR DESCRIPTION
**Pull request by StarlaStarla**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"
- [x] reviewers are notified about the pull request

### CODE REVIEW

**Review by ilya-asiyuk-epam — APPROVED** [review]

File: ketcher-autotests/tests/Templates/Functional-Groups/Functional-Group-Tools/functional-group-tools.spec.ts [hunk 1]
```diff
@@ -813,7 +813,6 @@ test.describe('Templates - Functional Group Tools3', () => {
     Description: Can attach copied Functional Group to atoms of structure
     Test not working proberly right now. Bug https://github.com/epam/ketcher/issues/2660
    */
-    test.fail();
```
Comment (rrodionov91) [inline ketcher-autotests/tests/Templates/Functional-Groups/Functional-Group-Tools/functional-group-tools.spec.ts:816]: This change is not related to the sequence selection feature. Why we decided to do it here? If it really needed is comment above the test still valid?

**Review by rrodionov91 — CHANGES_REQUESTED** [review]

File: ketcher-autotests/tests/Templates/Functional-Groups/Functional-Group-Tools/functional-group-tools.spec.ts (hunk 1 above)
Comment (StarlaStarla) [reply ketcher-autotests/tests/Templates/Functional-Groups/Functional-Group-Tools/functional-group-tools.spec.ts:816]: Because this test is valid, and this `test.fail();` will make the test fail. Sure, I will add comment.

**Review by rrodionov91 — APPROVED** [review]
